.gitignore

```diff
@@ -3,8 +3,8 @@
 *.vscode
 
 # Additional Ignore to exclude torchvision code and docs.
-docs/
-torchvision/
+/docs/
+/torchvision/
```

Comment:
Oooh my bad. I think I had added this in a PR.
Apologies :pray:

Reply:
This occurred when I was migrating docs from master to main branch. I thought the generated folder were unnecessary. But it doesn't seem to be the case. Sorry once again :(

Reply:
No problem @oke-aditya , if anything, it's our responsibility as reviewer to make sure nothing breaks, not yours.

This is the main reason why we are quite defensive when it comes new features, or seemingly harmless changes: even the smallest change can boomerang at us in ways we can't really imagine at the time of the PR. So sometimes, we push back on including something - it can be a bit frustrating for the contributor, but it can help avoiding a lot of headache in the long run.

Reply:
I will keep this in mind.